charts/datafold/charts/worker/templates/deployment.yaml

```diff
@@ -47,11 +47,60 @@ spec:
           env:
             {{ include "worker.env" . | nindent 12 }}
             {{ include "datafold.env" . | nindent 12 }}
+            - name: DATAFOLD_FRESHPAINT_FRONTEND_TOKEN
```

Comment:
Shall we make this conditional ?  The singletons deployment could set a variable in top values.yaml on the worker deploy, so that these variables are only inserted when that var is true. That way other workers who have nothing to do with it don't see the evn vars?